### ISSUE TITLE
False Positive on Improper Use of Link

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#1423]
I can't figure out why this would flag improper use of link:
```
<a href="https://highland.edu" tabindex="0" title="Link to the homepage.">
<img src="https://highland.edu/wp-content/themes/highland_edu/images/logo.jpg" alt="Highland Community College. Located in Northwest Illinois">
</a>
```

https://highland.edu/view-positions/?edac=3549113&edac_nonce=e8619b742e

There are other things wrong with it like the tabindex and the title attribute, but is does seem like a correct link.

**Comment by amberhinds** [comment]
Here's another example on the same page:
```
<a href="https://highland.edu/staff-directory/">Directory</a>
```
https://highland.edu/view-positions/?edac=3549114&edac_nonce=e8619b742e
